components/Button/Button.css

```diff
@@ -0,0 +1,132 @@
+@import url("../../style.css");
+@import url("../Avatar/Avatar.css");
```

Comment:
instead of importing avatar and color css here, import them in style css

Reply:
yea i have done that . thanks for pointing it out .